index.html

```diff
@@ -70,11 +67,11 @@ <h5 class="venue">
     <!-- main program section -->
     <section class="main-program">
       <h2 class="main-title">Main Program</h2>
-      <img src="#" alt="" />
+      <img src="#" alt="image" />
       <div class="mainCard">
         <div class="cardA">
           <div class="card-img">
-            <img src="image/mpro-img1-removebg-preview.png" alt="" />
+            <img src="image/mpro-img1-removebg-preview.png" alt="image" />
           </div>
```

Comment:
- [ ] please fix the broken link circled 🥂 
![image](https://user-images.githubusercontent.com/66230306/227712449-2e6eae73-d24a-4ebe-bd70-6efb6b08e152.png)